producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtInternalEndpointPortTypeTest.java

```diff
@@ -61,53 +67,50 @@ public class SdtInternalEndpointPortTypeTest extends AbstractSdtUnitTestBase {
     /**
      * Mocked IWsUpdateItemHandler instance.
      */
+    @Mock
     private IWsUpdateItemHandler mockUpdateItemHandler;
 
     /**
      * Set up common for all tests.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
-        portType = new SdtInternalEndpointPortType();
+        MockitoAnnotations.openMocks(this);

```

Comment:
I think the @ExtendWith(MockitoExtension.class) annotation initialises the mocked objects (see https://javadoc.io/static/org.mockito/junit-jupiter/2.20.0/org/mockito/junit/jupiter/MockitoExtension.html) so is this needed?

Reply:
The MockitoAnnotations.openMocks() command has been removed.